src/ducks/files/files.js

```diff
@@ -0,0 +1,2 @@
+// selectors
+export const getFiles = state => state.view.files
```

Comment:
Why do you divide the files duck into several tiny parts? We should think about logical responsibilities to organize files, functions and components.